README.md

```diff
@@ -98,6 +99,9 @@ end
 puts "There are now #{DmItem.count} rows in the digital media items table"
 ```
 
+## Usage
+To access JSON data via the API, requests should have the format `/api/dm_items/10?user_email=example@example.com&user_token=1G8_s7P-V-4MGojaKD7a`. The token is available in the app GUI.
```

Comment:
Is this the only endpoint that's available via the API? If not, you may want to say something like "API requests should include user email and token as URL parameters, for example..."

Reply:
Yes, right now that is the only endpoint available via the API